src/client/Microsoft.Identity.Client.Broker/WamAdapters.cs

```diff
@@ -183,16 +204,19 @@ private enum ResponseStatus : int
                     logger.Warning("No correlation ID in response");
                     correlationId = null;
                 }
+                
+                //parsing Pop token from auth header if pop was performed. Otherwise use access token field.
+                var token = authResult.IsPopAuthorization ? authResult.AuthorizationHeader.Split(' ')[1] : authResult.AccessToken;
```

Comment:
This behavior is a bit funky and not intuitive. I guess, why the difference? Should be documented a bit more (maybe here in code). Maybe changed on the interop side to be consistent? 
+@gladjohn